snuba/settings.py

```diff
@@ -113,6 +113,8 @@
 
 TOPIC_PARTITION_COUNTS: Mapping[str, int] = {}  # (topic name, # of partitions)
 
+ERRORS_ROLLOUT_WRITABLE_STORAGE: bool = True
```

Comment:
default to true ?

Reply:
oof, this is meant to be false. thanks for catching